makefile

```diff
@@ -267,6 +267,12 @@ mklgpufpk.HEADERS := $(MKLGPUFPKDIR.include)/mkl_dal_sycl.hpp $(MKLGPUFPKDIR.inc
 # Release library names
 #===============================================================================
 
+major_version := 1
+minor_version := 0
```

Comment:
minnor version is 2 for master branch

Reply:
I think it should be 1 for master as gold is 1.0

Reply:
major binary version = 1, minor = 0 (for gold), am I right? I will increment minor version for master after merge of this PR.